src/components/CountryDetails.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
```

Comment:
Same problem here, you're using useState and useEffect but not importing it.

`import React, {useState, useEffect} from 'react'`